bayesian/api/api_v2.py

```diff
@@ -307,6 +309,24 @@ def stack_analyses():
         raise HTTPError(500, e.args[0])
 
 
+@api_v2.route('/report/<stack_id>/<crda_key>', methods=['GET'])
```

Comment:
Can't we extract crda_key from stack_id from the RDS?

Reply:
Nope, we need a user reference even for RDS call

Reply:
why do you need user reference for rds access? just stack id is enough, thats how SA GET works.

Reply:
every stack id associated with user id.

Reply:
look at https://github.com/fabric8-analytics/fabric8-analytics-server/blob/44ab7d7ac248aeb2e14ae7f6fbc3a57e5e8e2ae6/bayesian/utils.py#L151.
it just needs request id.

Reply:
I don't feel we need a DB call for this purpose at all. 
All DB calls add to monetary costs which in this case could be avoided.

Reply:
The idea here is to make the url more concise. @prashbnair WDTY?

Reply:
I am fine with the current URL.